src/main/java/com/puppycrawl/tools/checkstyle/xpath/AbstractNode.java

```diff
@@ -358,5 +358,4 @@ public final void copy(Receiver receiver, int index, Location location) {
     private static UnsupportedOperationException throwUnsupportedOperationException() {
         return new UnsupportedOperationException("Operation is not supported");
     }
-
```

Comment:
Please undo this change. A blank line before type's closing brace is on purpose.

Reply:
done

Reply:
still not done.

Reply:
now it is done)